extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/devmode/ContinuousTestWebSocketHandler.java

```diff
@@ -24,24 +24,18 @@
         ContinuousTestingWebsocketListener.setStateListener(new Consumer<ContinuousTestingWebsocketListener.State>() {
             @Override
             public void accept(ContinuousTestingWebsocketListener.State state) {
-                StringBuilder sb = new StringBuilder();
-                //we don't have JSON support so we manually build it
-                //pretty yuck
-                //maybe we should try and move this to the deployment side somehow
-                sb.append("{\"running\":");
-                sb.append(state.running);
-                sb.append(", \"inProgress\":");
-                sb.append(state.inProgress);
-                sb.append(", \"run\":");
-                sb.append(state.run);
-                sb.append(", \"passed\":");
-                sb.append(state.passed);
-                sb.append(", \"failed\":");
-                sb.append(state.failed);
-                sb.append(", \"skipped\":");
-                sb.append(state.skipped);
-                sb.append("}");
-                lastMessage = sb.toString();
```

Comment:
This needs to be reverted. By default JSON support is excluded, so if don't have a JSON providing extension this will fail.

We use it for stuff on the deployment side, but not runtime.

Reply:
Hi Stuart. I am not using a Json framework, but an existing util class that does pretty much what you do here, that is available at runtime. It's in the same package as this class and are being use from other places in runtime, see https://github.com/quarkusio/quarkus/blob/main/extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/devmode/Json.java

Reply:
Ah, sorry. This looks good then.